src/Dialog/Dialog.js

```diff
@@ -49,13 +51,20 @@ class TransitionItem extends Component {
     this.setState({
       style: {
         opacity: 0,
-        transform: 'translate3d(0, 0, 0)',
+        transform: this.isTranslate3d() ? 'translate3d(0, 0, 0)' : 'translate(0, 0)',
       },
     });
 
     this.leaveTimeout = setTimeout(callback, 450); // matches transition duration
   }
 
+  isTranslate3d() {
```

Comment:
This is a utility function, not something that belongs on a component instance IMO. It's also poorly named, `isTranslate3d` sounds like we're asking something if it is `translate3d` which would only make sense if we were checking a style value. What is the function actually letting us know?